draft-vvv-webtransport-quic.md

```diff
@@ -403,30 +405,26 @@ a given client.
 The following entry is added to the "Application Layer Protocol Negotiation
 (ALPN) Protocol IDs" registry established by {{!RFC7301}}:
 
-The "wq-vvv-01" label identifies QUIC used as a protocol for WebTransport:
+The "wq-vvv-03" label identifies QUIC used as a protocol for WebTransport:
 
   Protocol:
   : QuicTransport
 
   Identification Sequence:
-  : 0x77 0x71 0x2d 0x76 0x76 0x76 0x2d 0x30 0x31 ("wq-vvv-01")
+  : 0x77 0x71 0x2d 0x76 0x76 0x76 0x2d 0x30 0x33 ("wq-vvv-03")
 
   Specification:
   : This document
 
-## Client Indication Fields Registry
+## Header Fields Registry
 
-IANA SHALL add a registry for "QuicTransport Client Indication Fields" registry.
+IANA SHALL add a registry for "QuicTransport Header Fields" registry.
```

Comment:
Could we instead use the HTTP headers registry?

Reply:
I'm a bit weary of that because the "Internet Message Header" registry is currently already overloaded, and from what I understand it's currently being reorganized, so I would wait here.

Reply:
I'm not sure if I understand why the reorganization of [the registry](https://www.iana.org/assignments/message-headers/message-headers.xhtml) would cause us to not use it